docs/manual/common/concepts/LagomDesignPhilosophy.md

```diff
@@ -9,7 +9,7 @@ Consider some of the basic requirements of a Reactive Microservice as identified
 
 The following Lagom characteristics promote these best practices:
 
-* Lagom is asynchronous by default --- its APIs make inter-service communication via streaming a first-class concept. All Lagom APIs use the asynchronous IO capabilities of [Akka Stream](https://akka.io/) for asynchronous streaming; the Java API uses [JDK8 `CompletionStage`](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletionStage.html)  for asynchronous computation; the Scala API uses [Futures](https://www.scala-lang.org/api/2.11.8/#scala.concurrent.Future).
+* Lagom is asynchronous by default --- its APIs make inter-service communication via streaming a first-class concept. All Lagom APIs use the asynchronous IO capabilities of [Akka Stream](https://akka.io/) for asynchronous streaming; the Java API uses [JDK8 `CompletionStage`](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletionStage.html)  for asynchronous computation; the Scala API uses [Futures](https://www.scala-lang.org/files/archive/api/2.12.4/scala/concurrent/Future.html).
```

Comment:
http://www.scala-lang.org/api/2.12.x/scala/concurrent/Future.html might be a slightly more future-proof link (2.12.x instead of 2.12.4) and we use it elsewhere in `ServiceImplementation.md`.